parlai/crowdsourcing/tasks/turn_annotations_static/webapp/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
```

Comment:
I'm not 100% sure but do we need __init__ under the webapp  @JackUrb ?

Reply:
Hmm, yeah, I don't think that's needed

Reply:
Done.